contact.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Product Landing Page</title>
+</head>
+
+<body>
+    <header id="header">
+        <nav id="nav-bar">
+            <ul id="navigation-bar"></ul>
+            <div>
+                <img id="header-img" src="gnysw2objzekbagrqiax.png" alt="Microverse">
+            </div>
+			<form action="index.html">
+			<ul>
+            <li><button type="button" class="nav-link">Home</button></li>
+			</ul>
+			</form>
+			<form action="contact.html">
+			<ul>
+            <li><button type="button" class="nav-link">Contact Us</button></li>
+			</ul>
+			</form>
+			<form action="review.html">
+			<ul>
```

Comment:
The code works perfectly. But, it would be better if the form elements with buttons used more appropriately. I advise the code to be more clean.